lib/stripe/stripe_client.rb

```diff
@@ -139,8 +139,9 @@ def execute_request(method, path,
       # stores information on the request we're about to make so that we don't
       # have to pass as many parameters around for logging.
       context = RequestLogContext.new(
+        account: headers["Stripe-Account"],
         api_key: api_key,
-        api_version: headers["Stripe-Version"] || Stripe.api_version,
```

Comment:
I realized that we already mix in `Stripe.api_version` when the headers get set above, so we don't need it to be checked here as well.